redbot/cogs/cleanup/cleanup.py

```diff
@@ -120,7 +120,8 @@ def message_filter(message):
                 if number is not None and number <= len(collected):
                     break
 
-        return collected
+        return collected, len(collected)
```

Comment:
Rather than returning `len(collected)` in the helper, just get the length from the returned result where you call this helper.